dubbo-common/src/main/java/org/apache/dubbo/config/AbstractInterfaceConfig.java

```diff
@@ -252,7 +252,7 @@
         }
         if (CollectionUtils.isNotEmpty(this.registries)) {
             this.registries.forEach(registryConfig -> {
-                if (registryConfig.getScopeModel() != applicationModel) {
+                if (registryConfig != null && registryConfig.getScopeModel() != applicationModel) {
```

Comment:
**Suggestion**: Adding a null check before accessing the 'getScopeModel()' method is a good practice to avoid potential NullPointerExceptions. <br> **Code Suggestion**: 
 ```
 The provided change correctly implements the suggested improvement.
 ```